server/src/test/java/org/apache/iotdb/db/sql/TqlParserTest.java

```diff
@@ -1,2021 +0,0 @@
-/*
```

Comment:
Write new tests, instead of removing the test class.

Reply:
TqlParserTest is not useful in antlr4, you can use Idea plugin to check the whether parser tree is correct.